src/megatron/bridge/training/utils/train_utils.py

```diff
@@ -458,13 +466,17 @@ def training_log(
                 writer.add_scalar(metric, value, iteration)
             if wandb_writer:
                 wandb_writer.log(throughput_report, iteration)
+            if mlflow_logger and throughput_report:
+                mlflow_logger.log_metrics(_sanitize_mlflow_metrics(throughput_report), step=iteration)
         if logger_config.log_memory_to_tensorboard:
             memory_report = report_memory(memory_keys=logger_config.memory_keys)
             memory_report = {f"memory/{mem_stat}": val for (mem_stat, val) in memory_report.items()}
             for metric, value in memory_report.items():
                 writer.add_scalar(metric, value, iteration)
             if wandb_writer:
                 wandb_writer.log(memory_report, iteration)
+            if mlflow_logger and memory_report:
```

Comment:
```suggestion
            if mlflow_logger:
```

For consistency with wandb logic.